setup.py

```diff
@@ -67,7 +67,7 @@ def find_version(*file_paths):
                 'gluoncv/nn/cython_bbox.pyx'
             ],
             extra_compile_args=[
-                '-Wno-cpp', '-O2'
+                '-Wno-cpp', '-O3'
```

Comment:
have you set cython as the dependency?

Reply:
it is an optional dependency. You don't have to use it